configure.py

```diff
@@ -1429,6 +1433,121 @@ def bakefile_test_sources(file, sources):
 
     f.close()
 
+def gen_cmake(build_paths, using_mods, cc, options):
+    def create_target_rules(sources):
+        target = {'sources': {}, 'frameworks': set(), 'libs': set()}
+        for source in sources:
+            target['sources'][source] = {'isa_flags': set()}
+        return target
+
+    def add_target_details(target, using_mod):
+        libs_or_frameworks_needed = False
+        for source_path in target['sources']:
+            for mod_source in using_mod.source:
+                if source_path == mod_source :
+                    libs_or_frameworks_needed = True
+                    for isa in using_mod.need_isa:
+                        target['sources'][source_path]['isa_flags'].add(cc.isa_flags[isa])
+        if libs_or_frameworks_needed == True:
+            if options.os in using_mod.libs:
+                for lib in using_mod.libs[options.os]:
+                    target['libs'].add(lib)
+            if options.os in using_mod.frameworks:
+                for framework in using_mod.frameworks[options.os]:
+                    target['frameworks'].add('"-framework %"' % framework)
```

Comment:
pylint just told me that there an "s" missing in this place: "-framework %" -> "-framework %s"

I am already fixing this and will submit erlier than expected because of this. But the version on master is not working